core/src/main/scala/com/databricks/labs/remorph/parsers/intermediate/functions.scala

```diff
@@ -271,7 +271,7 @@ class CallMapper extends IRHelpers {
     case CallFunction("TIMESTAMP_MILLIS", args) => MillisToTimestamp(args.head)
     case CallFunction("TIMESTAMP_SECONDS", args) => SecondsToTimestamp(args.head)
     case CallFunction("TO_CSV", args) => StructsToCsv(args.head, args(1))
-    case CallFunction("TO_DATE", args) => ParseToDate(args.head, args(1))
+    case CallFunction("TO_DATE", args) => ParseToDate(args.head, args.lift(1))
```

Comment:
what is lift, btw? take-as-optional?

Reply:
exactly, on a `Seq[A]`, `lift` returns a function of type `Int => Option[A]`.